### PR TITLE
Release/sp 2 andrii

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

**Comment by AntonSBilan** [comment]
@givehug it seems correct and working with strings, but we may want to test it before release to make sure all error messages are correct

File: lib/chains/elementChain.js [hunk 1]
```diff
@@ -89,7 +93,7 @@ const toString = (data, nameOnly = false) => {
 				break;
 			default: {
 				const matches = data.comparator.props.map(
-					one => `  ${getAllSymbols[one.name]} ${getAllSymbols[one.type]} ${getAllSymbols[one.val] || one.val}`
+					one => `  ${one.name} ${one.type} ${VALUE_M[one.val] || one.val}`
```
Comment (honeydripper) [inline lib/chains/elementChain.js]: This does not look correct (unless overwritten in later commits). The other line should have been taken.

Comment (givehug) [reply lib/chains/elementChain.js]: there are no symbols anymore, only VALUE

Comment (honeydripper) [reply lib/chains/elementChain.js]: right.
@AntonSBilan what about validationKeys.js? 

Not sure if some of those could appear in some of the output logs. If yes they will need to be translated.

**Comment by andriichumak** [comment]
@givehug, could you please resolve conflicts? We should have released this changes already.

File: lib/chains/pressButtonChain.js [hunk 2]
```diff
@@ -46,6 +44,7 @@ const toString = data => {
 	return out;
 };
 const beforeSendMsg = (data) => logger.log(getOpType(data), toString(data));
+// const beforeSendMsg = (data) => logger.info(chainIsAboutToBeExecuted(`pressButton(${data.ids.join(', ')})`, toString(data)));
```
Comment (andriichumak) [inline lib/chains/pressButtonChain.js]: Drop the commented line
```suggestion
```

File: lib/constants/borderStyle.js [hunk 3]
```diff
@@ -1,21 +1,23 @@
+/* eslint-disable */
```
Comment (andriichumak) [inline lib/constants/borderStyle.js]: We should not disable eslint completely, if necessary - disable specific rules. Same for other files below.

Comment (andriichumak) [reply lib/constants/borderStyle.js]: @givehug could you please as well fix this.

**Comment by givehug** [comment]
@andriichumak  merged

**Pull request merged**